tests/test_segment.py

```diff
@@ -12,7 +12,8 @@
 
 import numpy as np
 import scipy
-from scipy.spatial.distance import pdist, squareform
+from scipy.spatial.distance import cdist, pdist, squareform
+from nose.tools import raises
```

Comment:
we migrated away from nose and over to pytest.  Can you update the tests below accordingly?

Reply:
GH comments are not working so well, so I'll tack on here: why was .coveragerc deleted and recreated?  Can you revert that change?

Reply:
no idea what happened with .coveragerc... I guess it happened when I merged with the upstream master?